.github/actions/install-parcels/action.yml

```
@@ -18,14 +15,17 @@ runs:
       uses: al-cheb/configure-pagefile-action@v1.3
       with:
         minimum-size: 8GB
-    - name: Install micromamba (${{ inputs.environment-file }})
-      uses: mamba-org/setup-micromamba@v1
+    - name: Install miniconda (${{ inputs.environment-file }})
+      uses: conda-incubator/setup-miniconda@v3
       with:
         environment-file: ${{ inputs.environment-file }}
-        environment-name: ${{ inputs.environment-name }}
         channels: conda-forge
         cache-environment: true
         cache-downloads: true
+    - name: MPI support
+      if: ${{ ! runner.os == 'Windows' }}
+      run: conda install -c conda forge mpich mpi4py
```

Does conga-forge work without a dash?
```suggestion
      run: conda install -c conda-forge mpich mpi4py
```